docs/contributorfile.rst

```diff
@@ -132,75 +156,144 @@ Issues labels
+
+As we’re trying to maintain at least a 90% code coverage, you’re strongly encouraged to write all the necessary tests not to drop below the threshold. If our coverage becomes too low, you might be asked to add more tests and/or your PR might be rejected. See the `Automatic Testing <#testing>`_ section.
 
-As we’re trying to maintain at least a 90% code coverance, you’re strongly encouraged to write all the necessary tests not to drop below the threshold. If our coverance becomes too low, you might be asked to add more tests and/or your PR might be rejected.
+Don't merge your own pull request! That's a task for the main reviewer of your PR or the project manager. Remember that the project manager doesn't have to be a reviewer of your PR!
+
+.. _reviewing:
+
+Reviewing PRs
+-------------
+Reviewing PRs is an extremely important task in collaborative development. In fact, it is probably the task that requires the most time in the development, and it can be stressful for both the reviewer and the author. Remember that, as a PR Reviewer, you are granting that the codes works and integrates well with the rest of the repository, hence **you are responsible for the quality of the repository and its next version release**. If it doesn't integrate well, later PR reviewers might have to ask for broader changes than expected.
+There are many best practices to review code online, for instance `this one <https://medium.com/an-idea/the-code-review-guide-9e793edcd683>`_, but here are some good rules of thumbs that we need to follow while reviewing PRs:
+
+- Be respectful to the PR authors and be clear in what you are asking/suggesting - remember that, like you, they are contributing their spare time and doing their best job!
+- If there is a Draft PR, you can comment on its development in the message board or making "Comment" reviews. Don't ask for changes, and especially, **don't approve the PR**
+- If the PR graduated from Draft to full PR, check that it follows the sections `Pull requests <#pr>`_ and `Style Guide <#styling>`_ of these guidelines. If not, invite the author to do so before starting a review.
+- **Don't limit your review to the parts that are changed**. Look at the entire file, see if the changes fit well in it, and see if the changes are properly addressed everywhere in the code - in the documentation, in the tests, and in other functions. Sometimes the differences reported don't show the full impact of the PR in the repository!
+- Unless it's for typo fixes or similar, invite the author of the PR to make changes before actually doing them yourself. Request changes via comments or in the message board or by checking out the PR locally, making changes and then submitting a PR to the author's branch.
```

Comment:
To clarify, does this mean that reviewers shouldn't use the code suggestion tool (like you probably see throughout my review) or that they shouldn't directly commit to the contributor's branch? I personally like the code suggestion tool (obviously), so I'd like to allow reviewers to use it as needed.

Reply:
That's a practice I liked a lot and that I tried to pick up from @rmarkello: he never used the suggestion tools, opting for explaining the changes to the code he meant to apply and waiting for the PR author to do them instead - with the exception of rewording or typos or minimal things.
If you don't agree with this practice, we can reword it so that it's only against direct commits. I know that the suggestion tool make things waaaaaaaaaay faster, but for learning purposes is not the best.

Reply:
Maybe we could make an exception with newbies and make it educational for them. Or if one of us would like an extended description of the suggestion, we can just ask for it!

Reply:
To me this PR is ready to be merged once we figure this part out.

Reply:
I don't _think_ I have used the code suggestion tool before. I normally just make comments, or put in a PR to the author's branch, with my suggested changes. I guess it would be good to be clear about what we prefer, if anything.

Reply:
I'm going to rewrite this point in the next commit!